awx/main/dispatch/worker/base.py

```diff
@@ -128,7 +131,7 @@ def work_loop(self, queue, finished, idx, *args):
             if os.getppid() != ppid:
                 break
             try:
-                body = queue.get(block=True, timeout=1)
+                body = self.read(queue)
```

Comment:
does this change anything functional??

Reply:
no, looks like this change is just to accommodate the pattern in the callback-specific worker code

Reply:
Nope, this is just to persist the behavior for the base case (the dispatcher).